.github/workflows/pages.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and publish forkdiff github-pages
+permissions:
+  contents: write
+on:
+  push:
+    branches:
+      - optimism-history
+jobs:
+  deploy:
+    concurrency: ci-${{ github.ref }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1000  # make sure to fetch the old commit we diff against
+
+      - name: Build forkdiff
+        uses: "docker://protolambda/forkdiff:latest"
+        with:
+          args: -repo=/github/workspace -fork=/github/workspace/fork.yaml -out=/github/workspace/index.html
+
+      - name: Build pages
+        run: |
+          mkdir -p tmp/pages
+          mv index.html tmp/pages/index.html
+          touch tmp/pages/.nojekyll
+          if [ "$GITHUB_REPOSITORY" == "ethereum-optimism/op-geth" ]; then
+              echo "op-geth.optimism.io" > tmp/pages/CNAME
+          fi;
```

Comment:
This should do the trick, if `op-geth.optimism.io` is pointed at the github pages servers. And we need to enable pages in repo settings to point to the `gh-pages` branch when ready.